localtypings/pxtpackage.d.ts

```diff
@@ -48,6 +48,7 @@ declare namespace pxt {
         npmDependencies?: Map<string>;
         card?: CodeCard;
         additionalFilePath?: string;
```

Comment:
do we still need this one?

Reply:
Yes. User never sets `additionalFilePaths`, they only set `additionalFilePath`.